### PR TITLE
Add Paulo Felipe Souza' solution to toy app tutorial

### PR DESCRIPTION
**Pull request by Pauloparakleto**
Dear TOP developers,
Thanks a lot for this great project. Bellow, there is a description of all changes I suggest to the project.
I take the time to check all links in order to delete those whose path wore broken.
I also padronized all link names to be only "Heroku" stead of both "Heroku" and "Live".
At last, I remove a student solution which had submitted a **Hellow app** solution and a heroku link which shows the word "hello". This solution is there in chapter 1 from the RoR tutorial not in chapter 2. This change (removing a studant solution), I think is more delicated than the others so take a little close of it. Remember the command ask us to submit a solution for chapter 2.
Thanks!
May the force be with you.

### CODE REVIEW

File: rails_programming/introduction_to_rails/project_feet_wet.md [hunk 1]
```diff
@@ -22,51 +22,54 @@ In this chapter, you'll build a simple scaffolded app on Rails.  It won't be as
 </div>
 
 ### Student Solutions
-Send us your solution by editing this [file](https://github.com/TheOdinProject/curriculum/blob/master/rails_programming/introduction_to_rails/project_feet_wet.md) so we can show others! Submit a link to the Github repo with your files in it here using any of the methods listed on the [contributing page](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md).
+Send us your solution by editing this [file](https://github.com/TheOdinProject/curriculum/blob/master/rails_programming/introduction_to_rails/project_feet_wet.md) so we can show others! Submit a link to the Github repository with your files in it here using any of the methods listed on the [contributing page](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md).
```
Comment (105ron) [inline rails_programming/introduction_to_rails/project_feet_wet.md]: I know repository is probably better, but every other project uses `repo` and it's pretty commonly used so I think we should just keep it

File: rails_programming/introduction_to_rails/project_feet_wet.md [hunk 2]
```diff
@@ -22,51 +22,54 @@ In this chapter, you'll build a simple scaffolded app on Rails.  It won't be as
...
 
-If you will be submitting a solution for the RoR tutorial chapter 2, make sure it's in a repo that you won't overwrite when we move onto chapter 3 (which starts a new app)
+Submit your solution for the Ruby on Rails tutorial chapter 2, make sure it is in a repository that you will not overwrite when we move onto chapter 3 (which starts a new app).
```
Comment (105ron) [inline rails_programming/introduction_to_rails/project_feet_wet.md]: Agree with the full stop and RoR. But I would keep the contractions as it's standard English and commonly used. I think it's hard to read when you remove the contractions.

File: rails_programming/introduction_to_rails/project_feet_wet.md [hunk 3]
```diff
@@ -22,51 +22,54 @@ In this chapter, you'll build a simple scaffolded app on Rails.  It won't be as
...
 
 * Add your solution below this line!
-* [David Auza's and Luis Novoa's Solution](https://github.com/luis-novoa/rails_toy_app) - [Live](https://shrouded-island-61851.herokuapp.com/)
-* [Andrew Huntington's Solution](https://github.com/AndrewHuntington/toy_app) - [Live](https://toy-app-54058.herokuapp.com/)
-* [Nasser Abachi's Solution](https://github.com/abachi/toy-app) - [Live](https://abachidev-toy-app.herokuapp.com/)
-* [Brendaneus' Solution](https://theodinprojects.live/courses/ruby-on-rails/projects/toy-app)
-* [BShowen's Solution](https://github.com/BShowen/rails_toy_app) - [Live](https://bradley-toy-app.herokuapp.com)
-* [Vedant's Solution](https://github.com/vedantshetty/Odin_Project_Code/tree/master/Ruby%20On%20Rails/toy_app) - [Live](https://fast-scrubland-64335.herokuapp.com/users)
-* [Ian's solution](https://github.com/IanMKesler/rails_toy_app) - [Live](https://whispering-journey-28034.herokuapp.com)
+<<<<<<< HEAD
```
Comment (105ron) [inline rails_programming/introduction_to_rails/project_feet_wet.md]: Remove the merge conflict marker

File: rails_programming/introduction_to_rails/project_feet_wet.md [hunk 4]
```diff
@@ -22,51 +22,54 @@ In this chapter, you'll build a simple scaffolded app on Rails.  It won't be as
...
+* [Paulo Felipe Souza](https://github.com/Pauloparakleto/toy_app) - [Heroku](https://protected-woodland-57701.herokuapp.com/)
+* [David Auza's and Luis Novoa's Solution](https://github.com/luis-novoa/rails_toy_app) - [Heroku](https://shrouded-island-61851.herokuapp.com/)
+* [Andrew Huntington's Solution](https://github.com/AndrewHuntington/toy_app) - [Heroku](https://toy-app-54058.herokuapp.com/)
+* [Nasser Abachi's Solution](https://github.com/abachi/toy-app) - [Heroku](https://abachidev-toy-app.herokuapp.com/)
+* [Brendaneus' Solution](https://theodinprojects.Heroku/courses/ruby-on-rails/projects/toy-app)
+* [BShowen's Solution](https://github.com/BShowen/rails_toy_app) - [Heroku](https://bradley-toy-app.herokuapp.com)
+* [Vedant's Solution](https://github.com/vedantshetty/Odin_Project_Code/tree/master/Ruby%20On%20Rails/toy_app) - [Heroku](https://fast-scrubland-64335.herokuapp.com/users)
+* [Ian's solution](https://github.com/IanMKesler/rails_toy_app) - [Heroku](https://whispering-journey-28034.herokuapp.com)
+>>>>>>> 9aaafb64cc842b0568ce1e358992d1119a70c728
```
Comment (105ron) [inline rails_programming/introduction_to_rails/project_feet_wet.md]: remove the merge conflict marker

File: rails_programming/introduction_to_rails/project_feet_wet.md (hunk 2 above)
Comment (Pauloparakleto) [reply rails_programming/introduction_to_rails/project_feet_wet.md]: As a brazilian guy, It's nice to know about this standard English.

File: rails_programming/introduction_to_rails/project_feet_wet.md (hunk 3 above)
Comment (Pauloparakleto) [reply rails_programming/introduction_to_rails/project_feet_wet.md]: There is another one that I'll also remove.

File: rails_programming/introduction_to_rails/project_feet_wet.md (hunk 4 above)
Comment (Pauloparakleto) [reply rails_programming/introduction_to_rails/project_feet_wet.md]: This is the other one, thanks.

**Pull request closed without merging**